openerp/tools/config.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, fname=None):
         self.has_ssl = _check_ssl()
 
         self._LOGLEVELS = dict([
-            (getattr(loglevels, 'LOG_%s' % x), getattr(logging, x)) 
+            (getattr(loglevels, 'LOG_%s' % x), getattr(logging, x))
```

Comment:
This one-char change is on its own commit, so the chance of it conflicting is remote. 

I only added it because many editors tend to drop it automatically on save (mine included).

Anyway, I've dropped that commit. It is present in the original PR for Odoo, so if that gets merged eventually it will show up here.